<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefsDlg</class>
 <widget class="QDialog" name="PrefsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="startSchlGB">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Startup Schematic Folder</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QRadioButton" name="schLastRB">
          <property name="text">
           <string>Last Used Folder</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">schBG</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="schFolderRB">
            <property name="text">
             <string>Folder</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">schBG</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="schPathLE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>No Folder Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="schPathBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="startTmplGB">
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Startup Code Template</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QRadioButton" name="tmplDefaultRB">
          <property name="text">
           <string>Default Internal Template</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">tmplBG</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tmplLastRB">
          <property name="text">
           <string>Last Used Template</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">tmplBG</string>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="tmplFileRB">
            <property name="text">
             <string>File:</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">tmplBG</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tmplPathLE">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>No Template Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tmplPathBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="authorGB">
       <property name="title">
        <string>Template Text Substitution - Author</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QRadioButton" name="authorSystemRB">
            <property name="text">
             <string>System User Name</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">authBG</string>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QRadioButton" name="authorCustomRB">
              <property name="text">
               <string>Custom:</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">authBG</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="authorLE">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>No Author Text Entered</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="userFieldsGB">
       <property name="title">
        <string>Template Text Substitution - User Defined Fields</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="user1Lbl">
            <property name="text">
             <string>User Field 1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="user1TE">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Not Defined</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="user2Lbl">
            <property name="text">
             <string>User Field 2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="user2TE">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Not Defined</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="user3Lbl">
            <property name="text">
             <string>User Field 3:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="user3TE">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Not Defined</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrefsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrefsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="tmplBG"/>
  <buttongroup name="authBG"/>
  <buttongroup name="schBG"/>
 </buttongroups>
</ui>
